lib/MusicBrainz/Server/Data/Event.pm

```diff
@@ -150,20 +150,23 @@ sub load_meta
 }
 
 sub is_empty {
-    my ($self, $event_id) = @_;
-
+    my ($self, $event_id, $ignore_edits) = @_;
+
+    my $edits_pending_condition = 'AND edits_pending = 0' unless $ignore_edits;
+    my $open_edits_condition = 
+        'EXISTS (
+          SELECT TRUE
+          FROM edit_event JOIN edit ON edit_event.edit = edit.id
```

Comment:
I'm a bit confused about why some of these `JOIN edit` at all while others don't. Is there something I'm missing or should we standardise this while I'm at it?

Reply:
This seems to be required here to test `status` below.

Reply:
Oh. Some tables such as `edit_label` have a materialized `status` column, but some like `edit_event` do not. That explains this at least, but wtf, why?